app/assets/stylesheets/common/base/sidebar.scss

```diff
@@ -195,6 +200,10 @@
   .row-wrapper {
     display: grid;
     grid-template-columns: 2em 4.5em repeat(2, 1fr) 2em;
+
+    @include breakpoint(mobile-extra-large) {
+      grid-template-columns: 2em 4.5em repeat(2, 1fr);
+    }
```

Comment:
This triggers scss deprecation warning in `core system` CI job. (I'll add something to surface these better)

The solution is to move this `@include` after regular props, to line ~213